cordova-lib/src/plugman/registry/registry.js

```diff
@@ -236,15 +231,22 @@ function initSettings() {
     module.exports.settings =
     rc('plugman', {
         cache: plugmanCacheDir,
-        registry: 'http://registry.cordova.io',
+        registry: registryURL,
         logstream: fs.createWriteStream(path.resolve(plugmanConfigDir, 'plugman.log')),
         userconfig: path.resolve(plugmanConfigDir, 'config'),
         'cache-min': oneDay
     });
+
+    // if npm is true, use npm registry. 
+    // ~/.plugman/config overides the above config if it exists. 
```

Comment:
I think it's the same that Tim points out. Normally users don't have a registry override in their plugman/config, but when they do, we should use it. Probably the logic should actually be:

if (custom registry) {
  use it only
} else {
  try CPR and fallback to NPM
}

It's pretty fringe though, so maybe just add a TODO to support custom registries again?